data/styles/clear/include/Basemap.mapcss

```diff
@@ -33,6 +33,17 @@
 area[natural=coastline]
 {z-index: -10;}
 
+/* These are foreground areas with semi-transparent hardcoded hatching-style fills.
+   They are rendered over all other background areas only.
+   And we want lines to be rendered above them, hence minimum z-index.
+*/
+area[leisure=nature_reserve],
+area[boundary=national_park],
+area[landuse=military],
+area[landuse=military][military=danger_area],
+area[landuse=military][military=cordon]
```

Comment:
I suspect (not sure) that area[landuse=military] should be enough here.

Reply:
You're right!